src/parser/Parser.ts

```diff
@@ -592,8 +578,16 @@ export class Parser {
             }
         }
 
-        //consume the final `end interface` token
-        result.tokens.endEnum = this.consumeToken(TokenKind.EndEnum);
```

Comment:
`consumeToken` is not in `v1` !!
Also, this now has an error when it can't find `end enum`

Reply:
I added a test for this as well.

Reply:
ugh.. I think it was ME that removed `consumeToken`

I just reverted this change, because it should not be dealt with here